doc/bucket-class-crd.md

```diff
@@ -1,73 +1,85 @@
 [NooBaa Operator](../README.md) /
 # BucketClass CRD
+The BucketClass CRD represents a structure that defines bucket policies relating to data placement, [namespace](namespace-store-crd.md) properties, replication policies and more.
 
-NooBaaBucket CRD represents a class for buckets that defines policies for data placement, namespace, replication policy and more.
+Note that placement-bucketclass and namespace-bucketclass both use the same CR, and the difference lies inside the bucket class' `spec` section, more specifically the presence of either the `placementPolicy` or `namespacePolicy` key. 
 
-Data placement capabilities are built as a multi-layer structure, here are the layers bottom-up:
+# Definitions
+- CRD: [noobaa.io_bucketclasses_crd.yaml](../deploy/crds/noobaa.io_bucketclasses_crd.yaml)
+- CR: [noobaa.io_v1alpha1_bucketclass_cr.yaml](../deploy/crds/noobaa.io_v1alpha1_bucketclass_cr.yaml)
+
+## Placement Policy
+A placement bucket class defines a policy for standard buckets - i.e. NooBaa buckets that are backed by [backingstores](backing-store-crd.md).
+The data placement capabilities are built as a multi-layer structure, here are the layers bottom-up:
 - Spread Layer - list of backing-stores, aggregates the storage of multiple stores.
 - Mirroring Layer - list of spread-layers, async-mirroring to all mirrors, with locality optimization (will allocate on the closest region to the source endpoint), mirroring requires at least two backing-stores.
 - Tiering Layer - list of mirroring-layers, push cold data to next tier.
 
-Namespace policy:
-A namespace bucket-class will define a policy for namespace buckets.
-Namespace policy will require a type, the type's value can be one of the following: single, multi, cache.
-
+## Namespace Policy
+A namespace bucket class defines a policy for namespace buckets - i.e. NooBaa buckets that are backed by [namespacestores](namespace-store-crd.md).
```

Comment:
I'm confused... namespace policy is on noobaa buckets (encrypted, deduped, compressed) and it is not related to the namespace store (which saves the data plain)?